oh_staff_ui/tests.py

```diff
@@ -920,6 +963,40 @@ def test_valid_audio_item_mods(self):
 
         self.assertEqual(ohmods.is_valid(), True)
 
+    def test_valid_abstract_parse(self):
+        item = self.interview_item
+        ohmods = OralHistoryMods(item)
+        ohmods.populate_fields()
+
+        ohmods_from_string = load_xmlobject_from_string(
+            ohmods.serializeDocument(), mods.MODS
+        )
+
+        self.assertEqual(ohmods_from_string.is_valid(), True)
+
+        # Assert abstract generated properly
+        mods_xml = ohmods_from_string.serialize(pretty=True)
+        self.assertTrue(b"<mods:abstract>Abstract element</mods:abstract>" in mods_xml)
+
+    def test_valid_description_parse(self):
+        item = self.interview_item
+        ohmods = OralHistoryMods(item)
+        ohmods.populate_fields()
+
+        ohmods_from_string = load_xmlobject_from_string(
+            ohmods.serializeDocument(), mods.MODS
+        )
+
+        # Assert MODS still valid as read from string
+        self.assertEqual(ohmods_from_string.is_valid(), True)
+
+        # Assert specific description type properly exists
+        mods_xml = ohmods_from_string.serialize(pretty=True)
+        self.assertTrue(
+            b'<mods:note type="processinterview" displayLabel="Processing of Interview">'
+            in mods_xml
+        )
+
```

Comment:
If we are excluding admin notes, there should be a test which confirms the description added at line 913 above is *not* included in the output.

Reply:
Additional tests added for generic `note` and absence of `adminnote`